src/compas_viewer/components/renderer/camera.py

```diff
@@ -306,13 +331,13 @@ def _on_target_update(self, target: Position):
     def reset_position(self):
         """Reset the position of the camera based current view type."""
         self.target.set(0, 0, 0, False)
-        if self.renderer.viewmode == "perspective":
+        if self.viewer.renderer.viewmode == "perspective":
             self.rotation.set(pi / 4, 0, -pi / 4, False)
-        if self.renderer.viewmode == "top":
+        if self.viewer.renderer.viewmode == "top":
             self.rotation.set(0, 0, 0, False)
-        if self.renderer.viewmode == "front":
+        if self.viewer.renderer.viewmode == "front":
             self.rotation.set(pi / 2, 0, 0, False)
-        if self.renderer.viewmode == "right":
+        if self.viewer.renderer.viewmode == "right":
```

Comment:
You can also consider something like
```python
@property
def renderer(self):
    return self.viewer.renderer
```

You don't have to do it right here but can be useful when these path gets too long.